cloud-sql/postgres/knex/deployment.yaml

```diff
@@ -38,10 +38,8 @@ spec:
         env:
         - name: PORT
           value: "8080"
-        - name: INSTANCE_HOST
-          value: "127.0.0.1"
-        - name: DB_PORT
-          value: "5432"  
+        - name: INSTANCE_CONNECTION_NAME

```

Comment:
We shouldn't do this. This is an existing sample being used in a quickstart for "Deploying Cloud SQL Proxy with GKE" so changing this would break the quickstart, must be careful. We will eventually want to update this to use Connector but has to be a coordinated change across samples and docs.

Reply:
ok, with that in mind I'll reset all changes to yaml deployment files 😊 thanks!